.changeset/lovely-pianos-build.md

```diff
@@ -0,0 +1,22 @@
+---
+'astro': minor
+---
+
+Provides a new, experimental build cache for [Content Collections](https://docs.astro.build/en/guides/content-collections/). This includes multiple refactors to Astro's build process to optimize how Content Collections are handled, which should provide significant performance improvements for users with many collections.
+
+You can enable the build cache by adding the following flag to your Astro config:
```

Comment:
```suggestion
You can opt in to preview the new build cache by adding the following flag to your Astro config:
```
I think something like this implies a bit more that this is the way things will be in the future, and you get to opt in early, while it's experimental.

Reply:
Great suggestion, thank you!